### PR TITLE
[0.13.10] Fix bincompat issue introduced in #2268

### PR DESCRIPTION
**Pull request by eed3si9n**
Continuation of #2321
Ref #2268

/review @dwijnand, @Duhemm, @jsuereth

### CODE REVIEW

File: util/io/src/main/scala/sbt/IO.scala [hunk 1]
```diff
@@ -68,18 +86,29 @@ object IO {
   }
```
Comment (dwijnand) [inline util/io/src/main/scala/sbt/IO.scala]: Do you want to restore the semantics that were changed here as well?

https://github.com/sbt/sbt/commit/9fbbaab58e246bce7b21117e079206519c386b57?diff=unified#diff-dd0ad8518ad4dee1e8ab8f4f1d69cbe4R61

**Comment by dwijnand** [comment]
Ah I now understand what you meant in #2321 now...

Sorry for dropping the ball there and picking up the slack. Thanks, 1 comment.

File: util/io/src/main/scala/sbt/IO.scala (hunk 1 above)
Comment (eed3si9n) [inline util/io/src/main/scala/sbt/IO.scala]: I think the semantics for the `def classLocationFile(cl: Class[_]): File` remain the same.

Comment (dwijnand) [inline util/io/src/main/scala/sbt/IO.scala]: Looks to me like it used to fallback to using the system classloader (via old `classLocation(cl: Class[_]): URL`) and now it doesn't.

Comment (eed3si9n) [inline util/io/src/main/scala/sbt/IO.scala]: You're right. Do you want to take over porting the fixing on to this branch?

Comment (dwijnand) [inline util/io/src/main/scala/sbt/IO.scala]: I'm prepping to clean re-install my laptop so I can't ATM, sorry.

I will if this is still like this when I'm setup again (maybe this weekend maybe in 10 days..)

Comment (eed3si9n) [inline util/io/src/main/scala/sbt/IO.scala]: No problem. Added 068cffa57835530a91f8b798b482ef0a94200b33

**Comment by dwijnand** [comment]
Excellent, thank you. LGTM pending Travis.

**Pull request merged**